enterprise/server/cmd/ci_runner/main.go

```diff
@@ -970,7 +972,8 @@ func (ar *actionRunner) Run(ctx context.Context, ws *workspace) error {
 		artifactsDir := artifactsPathForCommand(ws, i)
 		namedSetID := filepath.Base(artifactsDir)
 
-		runErr := runCommand(ctx, *bazelCommand, expandEnv(args), action.Env, action.BazelWorkspaceDir, ar.reporter)
+		// Note: we don't use ar.action.Env here because it's already
```

Comment:
"it's already" what?

Reply:
lol, not sure how this creeped into this PR. Removed